js/forum/src/index.js

```diff
@@ -0,0 +1,26 @@
+import 'expose-loader?$!expose-loader?jQuery!jquery';
```

Comment:
What do these things do? 😕

Reply:
This [exposes](https://github.com/webpack-contrib/expose-loader) the jquery module as a global variable, so all existing core code would work without changes. Todo includes adding `import $ from 'jquery'` to the top of any file that uses it so we can potentially get rid of this.